docs/resources/registry_image.md

```diff
@@ -8,13 +8,10 @@ description: |-
 <!-- Bug: Type and Name are switched -->
 # docker_registry_image (Resource)
 
```

Comment:
https://github.com/kreuzwerker/terraform-provider-docker/pull/321 was reverted.

Reply:
How about updating Go code and generating the document again?

* https://github.com/kreuzwerker/terraform-provider-docker/blob/0da4706851ac8f86af531b5a1b725ae66e5b57af/internal/provider/resource_docker_registry_image.go#L11

Reply:
True, my bad. I missed this. I will update the go code and adapt to #321

Reply:
see https://github.com/kreuzwerker/terraform-provider-docker/pull/322/commits/5a0bbd78b60c64bf0878e927af88df4122e7038b @suzuki-shunsuke